Fix icons on footer

iramtahraoui opened the pull request:
- need help changing color on icons in footer
- color should change based on social media colors

iramtahraoui committed:
Fix icons on footer

- need help changing color on icons in footer
- color should change based on social media colors

rabah commented on index.html:
```diff
@@ -452,6 +452,7 @@ <h2>We're happy to hear from you</h2>
                     <li><a href="#">Android App</a></li>
                 </ul>
             </div>
+            <!--icons not changing color -->
```
Will check now

rabah replied:
@iramtahraoui i believe these changes are ready to merge so submit merge

rabah reviewed (commented):
Need to make the changes before submitting

Commit:
Fix hover icon colors

- Added specificity to css class for hover icons, so added ion-icon as the extra specific class.
- Corrected a tag placement in html footer

@iramtahraoui Should work fine now but double check
